collectors/nvd/collectors.py

```diff
@@ -210,6 +260,24 @@ def collect_updated(self) -> str:
             else ""
         )
 
+    @staticmethod
+    def create_snippet(item: dict) -> None:
+        """
+        Create a new snippet.
+        """
+        content = {
+            "cve_id": item["cve"],
+            "url": item["url"],
+            "description": item["description"],
+            "references": item["references"],
```

Comment:
I assume that we want to translate this into `FlawReference` list. Are we going to just default to external in the Snippet-Flaw translation? That might be OK. Taking the current options into account, it is the only meaningful. We could theoretically have an `NVD` reference type but I am not sure about the value and we can eventually retrieve it later if wanted.

Reply:
Updated as we discussed.